oer-booker2/src/App.js

```diff
@@ -26,16 +38,20 @@ class App extends Component {
       return (
         <div className="App">
           <h1>Home</h1>
-          <input type="text" placeholder="Search by title" />
-          <input type="button" value="Search" />
+          <input
```

Comment:
Might want to have the inputs in their own component. That would be a code clean thing.